content/es/firewall.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Firewall
```

Comment:
```suggestion
title: Cortafuegos
```

Reply:
@electrocucaracha  In my first review I suggested using the word `Firewall` instead of Cortafuegos. I think we discuss this in the meeting too.